### PR TITLE
Add pre-built binaries for cairo

### PR DESCRIPTION
**Pull request by uberhacker**
Tested on:
- [x] aarch64
- [x] armv7l
- [x] i686
- [x] x86_64

### CODE REVIEW

**Comment by JL2210** [comment]
Good job. Make sure to explicitly enable those dependencies, though.

**Comment by uberhacker** [comment]
Explicitly enable what?  All the dependencies are already inherited.

**Comment by JL2210** [comment]
Something like this:
```
           '--enable-lzo',
           '--enable-rsvg',
```

**Comment by uberhacker** [comment]
I believe those are already inheritedly enabled by default.

**Comment by JL2210** [comment]
Let me check that...

**Comment by JL2210** [comment]
Just enable them. They may not be enabled by default in the future, and it does no harm.
I don't think LZO is needed:
```
chronos@localhost ~/Downloads/tarballs/cairo-1.16.0 $ ./configure --help | grep -i lzo
chronos@localhost ~/Downloads/tarballs/cairo-1.16.0 $ 
```

**Comment by uberhacker** [comment]
Check your build log with and without lzo and librsvg dependencies.

File: packages/cairo.rb [hunk 1]
```diff
@@ -8,14 +8,22 @@ class Cairo < Package
   source_sha256 '5e7b29b3f113ef870d1e3ecf8adf21f923396401604bda16d44be45e66052331'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/cairo-1.16.0-chromeos-armv7l.tar.xz',
```
Comment (JL2210) [inline packages/cairo.rb]: This is good.

**Pull request merged**